source/sass/buyers-guide/components/header.scss

```diff
@@ -13,6 +13,7 @@ $category-nav-border-bottom-thickness: 4px;
 
   a {
     display: inline-block;
+    height: 42px;
```

Comment:
Ahh, deep dark CSS secret: Chrome rounds its pixel values, but Firefox does not, so in Chrome, this element is natively 42px, even if we don't set it. But in Firefox, the height is actually 42.5px and so it messes with the placement of the "face box": Firefox needs top:43px because of that 42.5px, but chrome needs 42px, and we can't set both. So instead, I'm fixing the height for these `a` to be 42px, which makes both firefox and chrome behave identically.

Reply:
:scream: thanks for the explanation